### DISCUSSION TITLE
how does skywalking trigger the tests

### DISCUSSION THREAD

**Discussion by taojintianxia** [apache/skywalking#8248] [Q&A] [answered]
as we heared, there are two repositories for skywalking, one for code, another one for test suit.
when a PR created in codebase, this will trigger the github action to launch some test suits.

what if the the test suits in another reposity can not receive any requst from the PR in skywalking codebase ?
I mean, how does skywalking luanch github action and trigger the github action in another repository ?

thanks for any response.

**Comment by wu-sheng** [comment, marked as answer]
Where do you hear? And who told you about this?

I am not sure what that, we definitely have multiple repositories, but we never did a test according a pull request from the other repository. I think you misread or misunderstand something.

**Reply by taojintianxia** [reply]
I see, thank you .
